Library/src/main/java/com/wunderlist/slidinglayer/transformer/SlideJoyTransformer.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * SlidingLayer.java
+ *
+ * Copyright (C) 2015 6 Wunderkinder GmbH.
+ *
+ * @author      Jose L Ugia - @Jl_Ugia
+ * @author      Antonio Consuegra - @aconsuegra
+ * @author      Cesar Valiente - @CesarValiente
+ * @version     1.2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.wunderlist.slidinglayer.transformer;
+
+import android.view.View;
+
+import com.wunderlist.slidinglayer.LayerTransformer;
+import com.wunderlist.slidinglayer.SlidingLayer;
+import com.wunderlist.slidinglayer.utils.Transitions;
+
+/**
+ * Created by joseluisugia on 16/03/15.
+ */
+public final class SlideJoyTransformer extends LayerTransformer {
+
+    @Override
+    public void transform(View layerView, float previewProgress, float layerProgress) {
+    }
+
+    @Override
+    protected void internalTransform(View layerView, float previewProgress, float layerProgress, int screenSide) {
+
+        float progressRatioToAnimate = Math.max(previewProgress, layerProgress);
+
+        // Scale
+        float scaleValue = Transitions.intermediateValueForCuePoints(progressRatioToAnimate,
+                new float[] { 0.9f, 1 });
+        layerView.setScaleX(scaleValue);
+        layerView.setScaleY(scaleValue);
+
+        // Rotation
+        float rotationX, rotationY;
+        float[] rotationXY = rotationValueForScreenSide(-4.75f, screenSide);
```

Comment:
Is it possible to mutate the object inside `rotationValueForScreenSide()` instead of creating new instances of `float[]` every time? Something like `rotationValueForScreenSide(float value, int screenSide, float[] result)`